site/docs/latest/getting-started/run-locally.md

```diff
@@ -1,16 +1,115 @@
 ---
-title: Run bookies locally
-prev: ../installation
+title: Run BookKeeper locally
 next: ../concepts
-toc_disable: true
 ---
 
-{% pop Bookies %} are individual BookKeeper servers. You can run an ensemble of bookies locally on a single machine using the [`localbookie`](../../reference/cli#bookkeeper-localbookie) command of the `bookkeeper` CLI tool and specifying the number of bookies you'd like to include in the ensemble.
+This tutorial guides you through every step of running BookKeeper locally.
 
-This would start up an ensemble with 10 bookies:
+## Install BookKeeper standalone
+
+Before installation, you need to check system requirements.
+
+### System requirement
+
+* [Unix environment](http://www.opengroup.org/unix)
+  
+* [Java Development Kit 1.6](http://www.oracle.com/technetwork/java/javase/downloads/index.html) or later
```

Comment:
Replace 1.6 with 8
I would link to https://adoptopenjdk.net as Oracle jdk is no more free for production usage

Reply:
@eolivelli sorry for the later reply, I saw your message just now.
Many thanks for your comment, I've incorporated it, PTAL.